File: Java/02-Tipos de Variáveis/Entrada.java
import java.util.Scanner;

/*Para receber comandos de entrada devemos importar a classe Scanner */

class Entrada {
    public static void main(String[] args) {
        /* Agora precisamos criar um objeto da classe Scanner */
        Scanner entrada = new Scanner(System.in);

        System.out.println("Digite seu nome : ");
        String nome = entrada.nextLine(); /* O nome digitado sera armazenado em uma variavel */

        System.out.println("Digite sua idade : ");
        int idade = entrada.nextInt(); /* O numero digitado sera armazenado em uma variavel */

        System.out.println("Valores digitados pelo usuario :");
        System.out.println(nome);
        System.out.println(idade);

        entrada.close(); /* Vamos fechar o Scanner */
    }
}

File: C/04-Entrada de Dados/Entrada.c
#include<stdio.h>

int main(void)
{
    //Para recebermos dados digitados pelo usuario usamos função scanf
    //|Vamos pedir para o usuario digitar um numero e mostra-lo
    int numero;

    printf("Digite um numero : ");
    //Numero inteiro usamos %d
    scanf("%d",&numero);
    //Vai armazenar o numero digitado na variavel declarada(não esquecer &)

    //Agora vamos mostrar
    printf("%d" , numero);
    return 0;
}
